### ISSUE TITLE
strange behaviour with .dockerignore

### ISSUE THREAD

**Issue by antonversal** [GoogleContainerTools/skaffold#386]
microservices example
### Expected behavior
should ignore files stated in `.dockerignore` without `**`, like `app.go`
### Actual behavior
only ignores files which have `**`, like `**/app.go`
### Information
Given microservices example
And `.dockerignore` file in same folder as `Dockerfile` with content `app.go`
When `skaffold dev` is run 
It does not ignore `app.go`
```
Skaffold version: 0.4.0
Operating system: macOS
Content of skaffold.yaml: 
apiVersion: skaffold/v1alpha2
kind: Config
build:
  artifacts:
  - imageName: gcr.io/k8s-skaffold/leeroy-web
    workspace: ./leeroy-web/
    docker: {}
  - imageName: gcr.io/k8s-skaffold/leeroy-app
    workspace: ./leeroy-app/
    docker: {}
  local: {}
deploy:
  kubectl:
    manifests:
    - ./leeroy-web/kubernetes/*
    - ./leeroy-app/kubernetes/*
```

### Steps to reproduce the behavior

  1. echo 'app.go' > leeroy-app/.dockerignore 
  2. skaffold dev should fail

**Comment by dgageot** [comment]
@antonversal this should be fixed in 0.6.0. Can you confirm?

**Comment by dgageot** [comment]
oh no. not merged... Forget about it ;-)
